packages/app/src/cli/models/extensions/specifications/app_config_app_proxy.ts

```diff
@@ -14,11 +15,18 @@ const AppProxySchema = zod.object({
 
 export const AppProxySpecIdentifier = 'app_proxy'
 
+const AppProxyTransformConfig: TransformationConfig = {
```

Comment:
I don't understand why transformation is being added here (and below) -- "these are needed so that no additional properties are accidentally included" -- what does this mean? Was something broken?

Reply:
If its doing something like filtering the submitted config, do we have test coverage demonstrating that?

Reply:
Its just these two modules, right?

Maybe making it mandatory for `createConfigExtensionSpecification` does the job?